proposals/0000-initial-extension-categorization.md

```diff
@@ -207,10 +200,16 @@ The following are the extensions that are uncategorized as of this proposal. The
 | Strict                  |
 | TemplateHaskell         |
 | TemplateHaskellQuotes   |
+| TransformListComp       |
 | Trustworthy             |
+| TupleSections           |
```

Comment:
It's somewhat surprising to see `TupleSections` in an uncategorized category, as it is included in `GHC2021` and `GHC2024`. Similarly `MonoLocalBinds`.

I'd hope they will be next on the list to discuss.

Reply:
FWIW,

```
Prelude> :set -XNoRelaxedPolyRec
<no location info>: warning: [GHC-53692] [-Wdeprecated-flags]
    -XNoRelaxedPolyRec is deprecated: You can't turn off RelaxedPolyRec any more
```

I guess it arguable that it may not be stable as whole extension flag could be removed. But it's an interesting special case.